src/bandersnatch/tests/plugins/test_storage_plugins.py

```diff
@@ -670,7 +670,7 @@ def test_iter_dir(self) -> None:
 
         self.assertListEqual(
             list(sorted(base_path.iterdir(), key=lambda p: str(p))),
-            list(sorted([elem[0] for elem in lists], key=lambda p: str(p))),
+            list(sorted((elem[0] for elem in lists), key=lambda p: str(p))),
```

Comment:
Just out of curiosity, what was the complaint here and from what tool? Sorting a list or tuple shouldn't differe here I thought :|

Reply:
Blame pyupgrade, I didn't understand it either but also couldn't find a way to turn it off from a quick scan :/